src/components/ec-dropdown/ec-dropdown.vue

```diff
@@ -11,22 +11,27 @@
     :disabled="disabled"
     :level="level"
     :is-loading="isLoading"
+    :popper-modifiers="popperModifiers"
+    :popover-options="popoverOptions"
     @change="onSelected"
     @open="$emit('open')"
   >
     <ec-input-field
       ref="trigger"
+      :id-from-parent="idFromParent"
       :value="selectedTextValue"
       :label="label"
       :error-message="errorMessage"
       :placeholder="placeholder"
       :disabled="disabled"
+      :icon-size="24"
       data-test="ec-dropdown__input"
       readonly
       icon="simple-arrow-drop-down"
-      :icon-size="24"
+      :is-in-group="isInGroup"
       @mousedown="onMousedown"
-      @focus="onFocus"
+      @focus="onFocus, $emit('onFocus', true)"
```

Comment:
1. emitted event should be in kebab-case
2. emitted event should not use prefix `on`.
3. don't pass true/false, use rather focus and blur events. It's not intuitive for any developer when you fire focus with false when input gets blurred. All of them are used to focus/blur events.
4. Don't put two statements into one single line. I noticed this trend in our code, it needs to stop! It makes templates very polluted, unclear and this logic should not be in templates.